<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: 
#FBF8F1
;</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
	background-color: #92A9BD; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
}

 QPushButton:hover {
	background-color: #7C99AC; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
 }</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="hasil_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Mulish&quot;;color:#000; </string>
   </property>
   <property name="text">
    <string>Waktu Proses:</string>
   </property>
  </widget>
  <widget class="QLabel" name="hasil">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Mulish&quot;;color:#000; </string>
   </property>
   <property name="text">
    <string>Plainteks:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Mulish&quot;;color:#1D2D5E; </string>
   </property>
   <property name="text">
    <string>Konversi File</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hasil_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Mulish&quot;;color:#000; </string>
   </property>
   <property name="text">
    <string>Ukuran File:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>351</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Mulish&quot;;color:#1D2D5E; 
height: 47.76695251464844px;
width: 376.5159912109375px;
left: 538px;
top: 229px;
border-radius: 13.25760269165039px;
border:2px solid #7C99AC;
padding-left:15px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="nilaip">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Mulish&quot;;color:#1D2D5E; 
height: 47.76695251464844px;
width: 376.5159912109375px;
left: 538px;
top: 229px;
border-radius: 13.25760269165039px;
border:2px solid #7C99AC;
padding-left:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Input P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="encryptB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
	background-color: #92A9BD; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
}

 QPushButton:hover {
	background-color: #7C99AC; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
 }</string>
   </property>
   <property name="text">
    <string>Enkripsi</string>
   </property>
  </widget>
  <widget class="QLabel" name="hasil_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Mulish&quot;;color:#000; </string>
   </property>
   <property name="text">
    <string>Cipherteks:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nilaiq">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Mulish&quot;;color:#1D2D5E; 
height: 47.76695251464844px;
width: 376.5159912109375px;
left: 538px;
top: 229px;
border-radius: 13.25760269165039px;
border:2px solid #7C99AC;
padding-left:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Input Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
	background-color: #92A9BD; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
}

 QPushButton:hover {
	background-color: #7C99AC; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
 }</string>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decryptB">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
	background-color: #92A9BD; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
}

 QPushButton:hover {
	background-color: #7C99AC; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
 }</string>
   </property>
   <property name="text">
    <string>Dekripsi</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Mulish&quot;;color:#000; </string>
   </property>
   <property name="text">
    <string>File Belum Diupload!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="genButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{ 
	background-color: #92A9BD; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
}

 QPushButton:hover {
	background-color: #7C99AC; color:#FFFFFF;
font: 17pt &quot;Mulish&quot;;color:#fff; 
	border-radius: 13.25760269165039px;
 }</string>
   </property>
   <property name="text">
    <string>Generate Key</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>backButton</zorder>
  <zorder>hasil_3</zorder>
  <zorder>hasil</zorder>
  <zorder>label_2</zorder>
  <zorder>hasil_4</zorder>
  <zorder>nilaip</zorder>
  <zorder>encryptB</zorder>
  <zorder>hasil_2</zorder>
  <zorder>nilaiq</zorder>
  <zorder>uploadButton</zorder>
  <zorder>decryptB</zorder>
  <zorder>fileName</zorder>
  <zorder>genButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
